app/app.go

```diff
@@ -372,8 +372,6 @@ func New(
 		authAddr,
 	)
 
-	logger.Info("bank keeper blocklist addresses", "addresses", app.BlockedAddrs())
-
```

Comment:
Shouldn't this be written to stderr? If not we should ensure the logger always writes to stderr.

Reply:
Why should we write informational log messages to stderr ? 
Also , is this log line used for anything ? We can consider exposing the list of BlockedAddreses via some other means if needed

Reply:
Typically you always write log messages to stderr to avoid situations just like this (where the command output is confused with log output)

It matters less for server processes but it matters a lot for cli tools

Reply:
```
func NewDefaultContext() *Context {
	return NewContext(
		viper.New(),
		tmcfg.DefaultConfig(),
		tmlog.NewTMLogger(tmlog.NewSyncWriter(os.Stderr)),
	)
}
```
is where the logger is configured to use stdout. This logic is called in cosmos-sdk `server.GetServerContextFromCmd` so it would be tricky to adjust.

Reply:
Okay, I get what you are saying. It might be useful to switch over to StdErr, but that would be out of scope for this pr. I created an issue; please add any comments that you can think of to the issue directly.
https://github.com/zeta-chain/node/issues/3046

For this PR, unless any one is consuming the log line, I would like to remove it for now.